packages/headless/src/modules/branch/services/index.ts

```diff
@@ -28,6 +28,14 @@ export class BranchService{
             errors: null,
             code: HttpStatus.BAD_REQUEST,
           }
+          
+        const urlExists = await this.branchRepo.getURL(branch.url);
```

Comment:
remove the function..you may use for this .
await this.branchRepo.getStore({url})